Makefile

```diff
@@ -94,6 +94,10 @@ update:
 	hack/make-rules/update.sh
 tidy:
 	hack/make-rules/tidy.sh
+# generate ip ranges
+generate-ipranges:
```

Comment:
nit: IMHO this is a misleading name, I'd expect generate to do what the `go generate ./...` bit does (generate code from inputs), not to download new data.

maybe `update-ipranges`